app/views/developers/developers.scala.html

```diff
@@ -12,19 +12,37 @@
 
 <article class="content__body full-width">
 
+    <style>
+        .developer-list tr {
+          word-break: break-all;
+        }
+
+        .input-select {
+           margin: 0;
+        }
+
+        .input-select--large {
+          width: 85%;
+        }
+
+        .developer-status {
+          min-width: 150px;
+        }
+    </style>
+
     <header>
         <h1>Dashboard</h1>
     </header>
 
     @tabList(activeTab = 1)
 
-    <form action="developers" method="post">
+    <form action="@routes.DevelopersController.developersPage(filter, status, users.pageNumber, users.pageSize).url" method="get">
```

Comment:
if the route mapping ever changed, then the reverse route would always work, whereas the hard-coded action would break